db/migrate/20200402174615_add_shark_tank_badge_to_badges.rb

```diff
@@ -1,5 +1,44 @@
 class AddSharkTankBadgeToBadges < ActiveRecord::Migration[5.2]
   def change
-    Badge.create({name: 'Shark Tank'})
+    Badge.find_or_create_by!({name: 'Shark Tank'})
```

Comment:
you can set this to 
`shark_tank_badge = Badge.find_or_create_by!({name: 'Shark Tank'})`